cves/kernel/CVE-2016-8630.yml

```
@@ -147,10 +156,17 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Searching through the bug report 
+    (https://bugzilla.redhat.com/show_bug.cgi?id=1393350) as well as the GitHub
+    (https://github.com/torvalds/linux/commit/d9092f52d7e61dd1557f2db2400ddb430e85937e)
+    and change log (https://mirrors.edge.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.8.7)
+    I could not determine how this vulnerability was discovered. It could have been 
+    detected early via a code-review or detected later without saying how the vulnerability
+    was discovered. There is not evidence either way.
```

Should be "There is no evidence either way."

But, consider a slight rewrite for clarity:

```suggestion
  answer: |
    The vulnerability's discovery method remains unclear after examining the bug
    report (https://bugzilla.redhat.com/show_bug.cgi?id=1393350), GitHub commit
    (https://github.com/torvalds/linux/commit/d9092f52d7e61dd1557f2db2400ddb430e85937e),
    and change log
    (https://mirrors.edge.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.8.7). It
    might have been detected early via code-review or found later without
    disclosure of the discovery method. No concrete evidence was found.
```